Gir.toml

```diff
@@ -1781,6 +1780,9 @@ manual_traits = ["TextBufferExtManual"]
 [[object]]
 name = "Gtk.TextIter"
 status = "generate"
+init_function_expression = "|_ptr| ()"
```

Comment:
That is really super ugly... Maybe we should think about an enum?

Reply:
How would that enum look like? We somehow need to provide a way to provide a closure here as there are too many different cases.